docs/faqs-and-tips.md

```diff
@@ -0,0 +1,21 @@
+---
+title: FAQs and Tips
+description: Learn about top tips from and common problems faced by the React Native Firebase community.
+next: /releases
+previous: /migrating-to-v6
```

Comment:
The `/releases` pages also needs the `previous` tag changing.

Reply:
Done!

Reply:
Thanks